lld/ELF/ScriptLexer.cpp

```diff
@@ -231,6 +231,23 @@ ScriptLexer::Token ScriptLexer::till(StringRef tok) {
   return {};
 }
 
+ScriptLexer::Token ScriptLexer::readNameTill(StringRef tok) {
+  // this behaves like till but expects that token to be an
+  // identify of quoted string
+  StringRef str = next();
+  if (str.starts_with("\""))
+    str = str.substr(1, str.size() - 2);
+  if (str == tok)
+    return {};
+  if (str == "(" || str == ")" || str == "}")
+    setError(tok + " is missing before " + str);
```

Comment:
For new diagnostics, use single quotes: `';' is missing before '}'`

Reply:
Fixed. Please let me know if its in the format you want to have.

Reply:
Wait this Commit accidentally included the token file so let me delete it first. Sorry for the inconvenience.

Reply:
Okay deleted